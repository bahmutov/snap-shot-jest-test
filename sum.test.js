const sum = require('./sum')
const snapshot = require('snap-shot')

test('adds 1 + 2 to equal 3', () => {
  const result = sum(1, 2)
  expect(result).toMatchSnapshot()
})

test('concat strings', () => {
  const result = sum('f', 'oo')
  expect(result).toMatchSnapshot()
})

test('2 + 5 should match snap-shot', () => {
  const result = sum(2, 5)
  snapshot(result)
})

test('concat strings using snap-shot', () => {
  const result = sum('f', 'oo')
  snapshot(result)
})
