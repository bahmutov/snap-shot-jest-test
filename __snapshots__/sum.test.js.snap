exports[`test adds 1 + 2 to equal 3 1`] = `3`;

exports[`test concat strings 1`] = `"foo"`;
